source/dub/commandline.d

```diff
@@ -136,6 +140,16 @@ int runDubCommandLine(string[] args)
 		else if (options.quiet) loglevel = LogLevel.warn;
 		else if (options.verror) loglevel = LogLevel.error;
 		setLogLevel(loglevel);
+
+		if (options.noColors && options.forceColors) {
+			logError("Incompatible options: --no-colors and --force-colors");
```

Comment:
i'd rather see `--color={on|off|force}`

- `on`: activated, however detect if DUB is piped
- `off`: deactivated.
- `force`: activated, don't care about piping (or whatever else is the semantic for that).

However i wont get mad if other do't agree.

Reply:
I agree the current implementation is not that great. What about `--colors={auto|on|off}`? With `auto` being the default

Reply:
yes that would be nicer.